src/main/java/de/retest/recheck/ui/descriptors/IdentifyingAttributes.java

```diff
@@ -184,18 +184,26 @@ public double match( final IdentifyingAttributes other ) {
 	@Override
 	public String toString() {
 		final String type = getType();
-		final String text = get( "text" );
 		if ( type == null ) {
 			return "";
 		}
 		String result = type;
 		if ( type.lastIndexOf( '.' ) > -1 ) {
 			result = type.substring( type.lastIndexOf( '.' ) + 1, type.length() );
 		}
+		final String text = get( "text" );
 		if ( text != null ) {
-			result += " [" + text + "]";
+			return result + " [" + text + "]";
 		}
-		return result;
+		final String id = get( "id" );
+		if ( id != null ) {
+			return result + " [" + id + "]";
+		}
```

Comment:
Isn't `id` web-specific? At least I can only find `name` in the `identifyingAttributes` key list above.

Reply:
It is indeed, but by using the `retestId` in the `Element`s `toString` method, we address this. So this should be now almost irrelevant...